CMakeLists.txt

```diff
@@ -20,8 +26,8 @@ add_library(gfx STATIC gfx.cpp)
 set(GFX_DXC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/third_party/dxc)
 set(GFX_PIX_PATH ${CMAKE_CURRENT_SOURCE_DIR}/third_party/WinPixEventRuntime)
 
-set(GFX_DXC_PATH ${GFX_DXC_PATH} PARENT_SCOPE)
-set(GFX_PIX_PATH ${GFX_PIX_PATH} PARENT_SCOPE)
+# set(GFX_DXC_PATH ${GFX_DXC_PATH} PARENT_SCOPE)
+# set(GFX_PIX_PATH ${GFX_PIX_PATH} PARENT_SCOPE)
```

Comment:
Any reason for commenting these out? I believe we rely on them in Capsaicin if I'm not mistaken.